server/integration/feature-pack/src/main/resources/content/bin/prometheus.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ -z "$JMX_CONF" ]
+then
+   if [ -z "$1" ]
+   then
+       echo "Neither JMX_CONF nor input parameters are defined. Exiting"
+       exit 1
+   else
+       JMX_CONF=$1
+   fi
+fi
+
+PROMETHEUS_JAR=$(ls $JBOSS_HOME/modules/system/add-ons/ispn/io/prometheus/jmx/ispn-9.4/*.jar | sed -e "s| |:|g")
+JBOSS_LOG_MANAGER_LIB=$(ls $JBOSS_HOME/modules/system/layers/base/org/jboss/logmanager/main/*.jar | sed -e "s| |:|g")
+WILDFLY_COMMON_LIB=$(ls $JBOSS_HOME/modules/system/layers/base/org/wildfly/common/main/*.jar | sed -e "s| |:|g")
```

Comment:
You cannot hardcode `ispn-9.4` anywhere as the code will break in 10.0. You need to use `@infinispan.module.slot.prefix@`

See https://github.com/infinispan/infinispan/commit/0756eea115e6d42a2d8a6e7eae85d110578995ec for an example

Reply:
Using `main` for logmanager and wildlfy-common should be fine as they are provided as part of the WFLY distribution.

Reply:
I did change ispn-9.4 to @infinispan.module.slot.prefix@ but then it resurfaced again :-(